ttnn/cpp/pybind11/operations/creation.hpp

```diff
@@ -331,11 +341,27 @@ void bind_empty_like_operation(py::module& module) {
 
 void py_module(py::module& module) {
     detail::bind_full_operation(module, ttnn::full);
-    detail::bind_full_operation_with_hard_coded_value(module, ttnn::zeros);
+    detail::bind_full_operation_with_hard_coded_value(module, ttnn::zeros,
+    R"doc(Supported dtypes, layouts, and ranks:
+
+        +----------------------------+---------------------------------+-------------------+
+        |     Dtypes                 |         Layouts                 |     Ranks         |
+        +----------------------------+---------------------------------+-------------------+
+        |    BFLOAT16                |       ROW_MAJOR, TILE           |      2, 3, 4      |
+        +----------------------------+---------------------------------+-------------------+
+        |    BFLOAT_8                |          TILE                   |      2, 3, 4      |
+        +----------------------------+---------------------------------+-------------------+)doc");
     detail::bind_full_operation_with_hard_coded_value(module, ttnn::ones);
 
     detail::bind_full_like_operation(module, ttnn::full_like);
-    detail::bind_full_like_operation_with_hard_coded_value(module, ttnn::zeros_like);
+    detail::bind_full_like_operation_with_hard_coded_value(module, ttnn::zeros_like,
+    R"doc(Supported dtypes, layouts, and ranks:
+
+        +----------------------------+---------------------------------+-------------------+
```

Comment:
@ruthreshx if Zeros is not supporting the Row Major kindly update the layout column in the table